src/main.rs

```diff
@@ -757,9 +757,7 @@ fn run() -> Result<(), failure::Error> {
         let mut target = manifest.get_target(env, is_preview)?;
         let deploy_config = manifest.deploy_config(env)?;
 
-        let verbose = matches.is_present("verbose");
```

Comment:
removed this bc the reason for the flag made  sense when we were printing all of the file names that were being uploaded; now we just use a progress bar so we should just show it all the time and make the verbose silent.